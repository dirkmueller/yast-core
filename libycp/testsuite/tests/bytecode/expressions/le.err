{
    // filename: "tests/bytecode/expressions/le.ycp"
    return (42 <= 69);
}
