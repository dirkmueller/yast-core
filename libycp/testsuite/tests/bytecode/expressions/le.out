{
    return (42 <= 69);
}
